lib/Helper/Helper.cpp

```diff
@@ -34,9 +35,17 @@ char *fgets_stdio_blocking(char *str, int n) {
   return str;
 }
 
-void xSemaphoreTakeT(QueueHandle_t mutex) {
+void xSemaphoreTakeT(xQueueHandle mutex) {
   if (!xSemaphoreTake(mutex, portMAX_DELAY)) {
     cout << "ERROR: mutex ************************************ " << mutex << " ****************" << endl;
     throw runtime_error("ERROR: mutex");
   }
 }
+
+std::string getTimeStamp(unsigned long seconds) {
+  int secsRemaining = seconds % 3600;
+  int runHours = seconds / 3600;
+  int runMinutes = secsRemaining / 60;
```

Comment:
We might not run into performance issues for now (and it might even be that the compiler catches this). However modulo operations are generally very expensive (just as a side-note to keep in mind)